packages/expo-ui/components/ColorPicker/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { requireNativeView } from 'expo';
+import { useCallback } from 'react';
+import { NativeSyntheticEvent, processColor, StyleProp, ViewStyle } from 'react-native';
+
+/**
+ * Props for the ColorPicker component.
+ */
+export type ColorPickerProps = {
```

Comment:
could we maybe support `color` prop? (as in the tint of the actual picker button), or does it do nothing here?

Reply:
It looks like you aren't able to customize that in swiftui